pkg/network/controllers/vm_test.go

```diff
@@ -278,6 +278,28 @@ var _ = Describe("VM Network Controller", func() {
 		Expect(updatedVMI.Spec.Domain.Devices.Interfaces).To(Equal(updatedVM.Spec.Template.Spec.Domain.Devices.Interfaces))
 	})
 
+	It("sync succeeds to clear hotunplug interfaces from stopped VM", func() {
+		clientset := fake.NewSimpleClientset()
+		c := controllers.NewVMController(clientset)
+		unpluggedIface := libvmi.InterfaceDeviceWithBridgeBinding("foonet")
+		unpluggedIface.State = v1.InterfaceStateAbsent
+		vmi := libvmi.New(
+			libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+			libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			libvmi.WithInterface(unpluggedIface),
+			libvmi.WithNetwork(libvmi.MultusNetwork("foonet", "foonet-nad")),
+		)
+		vm := libvmi.NewVirtualMachine(vmi.DeepCopy())
+
+		originalVM := vm.DeepCopy()
+		updatedVM, err := c.Sync(vm, nil)
+		Expect(err).NotTo(HaveOccurred())
+
+		originalVM.Spec.Template.Spec.Networks = originalVM.Spec.Template.Spec.Networks[:1]
+		originalVM.Spec.Template.Spec.Domain.Devices.Interfaces = originalVM.Spec.Template.Spec.Domain.Devices.Interfaces[:1]
+		Expect(updatedVM).To(Equal(originalVM))
```

Comment:
Please consider the following alternative:

```suggestion
Expect(updatedVM.Spec.Template.Spec.Networks).To(Equal(originalVM.Spec.Template.Spec.Networks[:1]))
		Expect(updatedVM.Spec.Template.Spec.Domain.Devices.Interfaces).To(Equal(originalVM.Spec.Template.Spec.Domain.Devices.Interfaces[:1]))
```

Reply:
but this way we won't know if something else was changed (and nothing else should be changed)
also i adhered to what done in the test above L245
better that they will be the same

Reply:
we dont need it anymore in that test, the only benefit is syntactic as it is called `originalVM` but imho it doesn't merit a new variable, because this is the end of the test